UnityProject/Assets/Scripts/UI/Systems/Lobby/CharacterCustomization.cs

```diff
@@ -60,7 +60,8 @@ public class CharacterCustomization : MonoBehaviour
 
 		public CharacterDir currentDir;
 
-		[SerializeField] public List<Color> availableSkinColors = new List<Color>();
+		public List<Color> SkinColors;
```

Comment:
I don't understand this change

Reply:
it's supposed to load the skin colours from the race SO, or pop up the colour picker if there isn't any set

Reply:
> it's supposed to load the skin colors from the race SO, or pop up the color picker if there isn't any set

It had empty values, also no, it doesn't load anything from anywhere, you just set the color values yourself.
And if you read the comment in start() you know why I did this. These colors get deleted after the game loads so I had to store them somewhere.

Reply:
why public tho, why not make it so it doesn't delete the colors instead of making yet another list that will feed the previous list.

Reply:
> why public tho, why not make it so it doesn't delete the colors instead of making yet another list that will feed the previous list.

Would be handy if there was a way to actually find how that list is getting deleted in the first place to avoid this issue.

Reply:
yes, that means more investigation and debugging is required before calling the issue solved.